### DISCUSSION TITLE
Timezone

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#588] [Q&A (Help)] [answered]
I've searched on  help "timezone" but there's no result
I've seen in config/app.php this:
    /*
    |--------------------------------------------------------------------------
    | Application Timezone
    |--------------------------------------------------------------------------
    |
    | Here you may specify the default timezone for your application, which
    | will be used by the PHP date and date-time functions. We have gone
    | ahead and set this to a sensible default for you out of the box.
    |
    */

    'timezone' => 'Europe/Rome',

I need some advice, I'm making an international app that can be used by customers from all over the world, so I've decided to save all the datetimes in UTC format and however I have to adjust the backend so that it shows them with the correct timezone of the 'user, timezone which I can get from the browser or from a user configuration.
How can I best manage this?

**Comment by jcastroa87** [comment, marked as answer]
Hello @blondie63 

This is not a feature support by Backpack, in any case you can try, saving user timezone in they profile and manage laravel in UTC, then you need to play with Carbon to transform database time data to the user timezone.

Cheers.

**Reply by blondie63** [reply]
@jorgetwgroup Ok, how does the parameter app.php-> 'timezone' => '...'  affect the whole application?

**Reply by jcastroa87** [reply]
When you will get current time() with PHP you will get as UTC is you set as this.

**Reply by blondie63** [reply]
@jorgetwgroup But can i specify a timezone for CRUDs ? because if i've all db data as UTC but if i know that some users have different timezone i need to set it before show data..

**Reply by jcastroa87** [reply]
Nop, backpack have not this feature implemented, as i told you first you need to handle to convert from UTC (database save data) to user timezone and show it.

**Comment by blondie63** [comment]
@jorgetwgroup Hi, i've another question, BackPack is supporting ISO 8601 for timestamps and datetime fields ?
because i've tried but i always save date without offset..
